mobile-widgets/qml/main.qml

```diff
@@ -165,6 +165,8 @@ Kirigami.ApplicationWindow {
 	globalDrawer: Kirigami.GlobalDrawer {
 		id: gDrawer
 		height: rootItem.height
+		enabled: (prefs.credentialStatus === CloudStatus.CS_NOCLOUD ||
+			 prefs.credentialStatus === CloudStatus.CS_VERIFIED)
```

Comment:
this is a useful change. It has nothing to do with this PR.

Reply:
Your are totally correct, it is part of the 3 PR´s that provides a more uniform login (e.g. the user cannot escape, without having chosen how to load the divelst).

I apologize ! it should have been in newlogin3, I wanted to make the 3 login PR´s as one, because they really work together, but split it in 3 to keep the number of commits in each low.

I can move it, if you really want it, but it need to be in one of the 3 PRs because it is a fundamental limitation. Please advice ?

Reply:
I think this change was in one of the other PRs that I just merged. Or am I starting to confuse myself?